Index: _source/plugins/clipboard/plugin.js
===================================================================
--- _source/plugins/clipboard/plugin.js	(revision 7383)
+++ _source/plugins/clipboard/plugin.js	(revision 7382)
@@ -231,6 +231,7 @@
 			if ( mode == 'text' && CKEDITOR.env.gecko )
 				editor.document.getBody().focus();
 
+			pastebin.remove();
 			editor.removeListener( 'selectionChange', cancel );
 
 			// Grab the HTML contents.
@@ -243,9 +244,7 @@
 						 && ( bogusSpan.is && bogusSpan.hasClass( 'Apple-style-span' ) ) ?
 							bogusSpan : pastebin );
 
-			// IE7: selection must go before removing paste. (#8691)
 			sel.selectBookmarks( bms );
-			pastebin.remove();
 			callback( pastebin[ 'get' + ( mode == 'text' ? 'Value' : 'Html' ) ]() );
 		}, 0 );
 	}
Index: CHANGES.html
===================================================================
--- CHANGES.html	(revision 7383)
+++ CHANGES.html	(revision 7382)
@@ -55,7 +55,6 @@
 		<li><a href="http://dev.ckeditor.com/ticket/7932">#7932</a> : [IE] Fix click below content region scrolls page to top.</li>
 		<li><a href="http://dev.ckeditor.com/ticket/8630">#8630</a> : HTML events disabled when loading data.</li>
 		<li><a href="http://dev.ckeditor.com/ticket/8433">#8433</a> : [IE9] Use W3C event model.</li>
-		<li><a href="http://dev.ckeditor.com/ticket/8691">#8691</a> : [IE] Fix paste invalid HTML crashes browser.</li>
 	</ul>
 	<h3>
 			CKEditor 3.6.2</h3>
